doc/reference/files/config/jobs.rst

```diff
@@ -0,0 +1,15 @@
+jobs
+----
+
+Maximum number of concurrent jobs Dune is allowed to have.
```

Comment:
```suggestion
Maximum number of concurrent jobs Dune is allowed to have.
```
This is an incomplete sentence. Perhaps "There is a maximum number..." or "<some number> is the maximum..." or "`jobs` designates the maximum number..."